electrum/gui/kivy/uix/screens.py

```diff
@@ -341,7 +341,9 @@ def do_pay(self):
     def do_pay_invoice(self, invoice):
         if invoice.is_lightning():
             if self.app.wallet.lnworker:
-                self.app.protected(_('Pay lightning invoice?'), self._do_pay_lightning, (invoice,))
+                amount_sat = invoice.amount_msat/1000
+                msg = _("Pay lightning invoice?") + '\n\n' + _("This will send {}?\n").format(self.app.format_amount_and_units(amount_sat))
```

Comment:
Note that in kivy (unlike the similarly named method in qt), `format_amount_and_units` does not append the fiat value, so e.g. this will only return "0.05 BTC".
I suggest introducing a new method in main_window.py, named e.g. `format_amount_and_units_with_fiat` that behaves like `format_amount_and_units` does in qt: it returns a string with both the formatted btc value and the fiat value. (later, but not in this PR, all these `format_amount_*` method should be refactored amd cleaned up...)

Reply:
Please move the line break outside the translation, i.e.:
```suggestion
                msg = _("Pay lightning invoice?") + '\n\n' + _("This will send {}?").format(self.app.format_amount_and_units(amount_sat)) + '\n'
```